common/folderCreationTracker.go

```diff
@@ -63,18 +68,27 @@ func (f *simpleFolderTracker) RecordCreation(folder string) {
 	f.contents[folder] = struct{}{}
 }
 
-func (f *simpleFolderTracker) ShouldSetProperties(folder string, overwrite OverwriteOption) bool {
+func (f *simpleFolderTracker) ShouldSetProperties(folder string, overwrite OverwriteOption, prompter prompter) bool {
 	switch overwrite {
 	case EOverwriteOption.True():
 		return true
-	case EOverwriteOption.Prompt(), // "prompt" is treated as "false" because otherwise we'd have to display, and maintain state for, two different prompts - one for folders and one for files, since its too hard to find wording for ONE prompt to cover both cases. (And having two prompts would confuse users).
-		EOverwriteOption.IfSourceNewer(), // likewise "if source newer" is treated as "false"
+	case EOverwriteOption.Prompt(),
+		EOverwriteOption.IfSourceNewer(), // TODO discuss if this case should be treated differently than false
```

Comment:
Shouldn't IfSoucreNewer apply to folder updates too? It's a bit odd that folders are treated differently here, no?

Reply:
It's because (if I recall correctly) for folders we move _properties_, but the LMTs we currently get are not date of _properties_ last changed.  They are (for at least some sources), date folder children were added or removed.